big_tests/tests/distributed_helper.erl

```diff
@@ -62,7 +62,48 @@ rpc(Node, M, F, A) ->
 
 rpc(Node, M, F, A, TimeOut) ->
     Cookie = ct:get_config(ejabberd_cookie),
-    escalus_ct:rpc_call(Node, M, F, A, TimeOut, Cookie).
+    escalus_rpc:call(Node, M, F, A, TimeOut, Cookie).
+
+%% @doc Require nodes defined in `test.config' for later convenient RPCing into.
+%%
+%% The use case would be to require and import the same names in your suite like:
+%%
+%%  -import(distributed_helper, [mim/0, fed/0,
+%%                               require_rpc_nodes/1,
+%%                               rpc/4]).
+%%
+%%  ...
+%%
+%%  suite() ->
+%%      require_rpc_nodes([mim, fed]) ++ escalus:suite().
+%%
+%%  ...
+%%
+%%  example_test(_Config) ->
+%%      RPCResult = rpc(mim(), remote_mod, remote_fun, [arg1, arg2]),
+%%      ...
+%%
+require_rpc_nodes(Nodes) ->
```

Comment:
I'm against hiding IO in a function which is now completely declarative. We will know if a node is down anyway, because the RPC calls now throw instead of just returning an error by value and we will know with better granularity (the specific file, test and line).